src/wfs/src/test/java/org/geoserver/wfs/TransactionListenerTester.java

```diff
@@ -5,14 +5,14 @@
  */
 package org.geoserver.wfs;
 
-import java.util.ArrayList;
+import java.util.LinkedList;
```

Comment:
Why this change?

Reply:
An ArrayList stores in order of insertion just like LinkedList.
You must be confusing it with the HashMap/LinkedHashMap case.

Reply:
Sorry @aaime, change reverted